### PR TITLE
calling lifecycle methods

### PR DESCRIPTION
**Pull request by madmike80**

### CODE REVIEW

File: src/components/Item/Item.js [hunk 1]
```diff
@@ -6,22 +6,58 @@ import DeleteIcon from '@material-ui/icons/Delete';
...
-    })}
-  >
+class Item extends React.Component {
+  
+  componentDidMount() {
+    console.log('componentDidMount');
+  }
+  
+  componentDidUpdate() {
+    console.log('componentDidUpdate');
```
Comment (KsuBurn) [inline src/components/Item/Item.js:16]: Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/gh/WebHeroSchool/tr-todo/pullRequest?prid=6309890)

**Comment by KsuBurn** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/WebHeroSchool/tr-todo/pullRequest?prid=6309890&bid=20517126)

File: src/components/Item/Item.js [hunk 2]
```diff
@@ -6,22 +6,58 @@ import DeleteIcon from '@material-ui/icons/Delete';
...
+  }
+  
+  componentWillUnmount() {
+    console.log('componentWillUnmount');
```
Comment (KsuBurn) [inline src/components/Item/Item.js:20]: Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/gh/WebHeroSchool/tr-todo/pullRequest?prid=6309890)

File: src/components/Item/Item.js [hunk 3]
```diff
@@ -6,22 +6,58 @@ import DeleteIcon from '@material-ui/icons/Delete';
...
-  <li
-    className={classnames({
-      [styles.item]: true,
-      [styles.done]: isDone,
-    })}
-  >
+class Item extends React.Component {
+  
+  componentDidMount() {
+    console.log('componentDidMount');
```
Comment (KsuBurn) [inline src/components/Item/Item.js:12]: Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/gh/WebHeroSchool/tr-todo/pullRequest?prid=6309890)

**Comment by KsuBurn** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/WebHeroSchool/tr-todo/pullRequest?prid=6309890&bid=20517126)

File: src/components/Item/Item.js [hunk 4]
```diff
@@ -6,22 +6,41 @@ import DeleteIcon from '@material-ui/icons/Delete';
...
+  }
+  
+  componentDidUpdate() {
+    console.log('componentDidUpdate');
```
Comment (KsuBurn) [inline src/components/Item/Item.js:16]: Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/gh/WebHeroSchool/tr-todo/pullRequest?prid=6309890)

File: src/components/Item/Item.js [hunk 5]
```diff
@@ -6,22 +6,41 @@ import DeleteIcon from '@material-ui/icons/Delete';
...
+  }
+  
+  componentWillUnmount() {
+    console.log('componentWillUnmount');
```
Comment (KsuBurn) [inline src/components/Item/Item.js:20]: Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/gh/WebHeroSchool/tr-todo/pullRequest?prid=6309890)

File: src/components/Item/Item.js [hunk 6]
```diff
@@ -6,22 +6,41 @@ import DeleteIcon from '@material-ui/icons/Delete';
...
-  <li
-    className={classnames({
-      [styles.item]: true,
-      [styles.done]: isDone,
-    })}
-  >
+class Item extends React.Component {
+  
+  componentDidMount() {
+    console.log('componentDidMount');
```
Comment (KsuBurn) [inline src/components/Item/Item.js:12]: Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/gh/WebHeroSchool/tr-todo/pullRequest?prid=6309890)

**Comment by MaxKortelov** [comment]
Методы жизненного цикла - codacy ругалось на использование console.log в JS

**Pull request merged**